.github/workflows/local_example.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: gh-hosted-runners-16cores-1-24.04
     strategy:
       matrix:
-        topo: [consul,etcd,zk2]
```

Comment:
Was this removed because tests were failing when consul was used? AFAIK, consul support is roughly equivalent to ZooKeeper support.

Reply:
Sorry, I updated the issue but not this PR:

> It looks like consul is actually broken. It looks like it has been [deprecated for a while](https://vitess.io/docs/22.0/user-guides/configuration-basic/global-topo/), so I am going to try removing it from the local_example instead.It looks like consul is actually broken. It looks like it has been [deprecated for a while](https://vitess.io/docs/22.0/user-guides/configuration-basic/global-topo/), so I am going to try removing it from the local_example instead.